src/dcqc/main.py

```diff
@@ -204,3 +205,15 @@ def qc_file(
     report = JsonReport()
     suite_json = report.generate(suite)
     json.dump(suite_json, sys.stdout, indent=2)
+
+
+@app.command()
+def update_csv(
```

Comment:
This is a great addition! Can you spend a tiny amount of time looking into how to write to nextflow tower reports? (That can be a separate ticket, but if it's super easy, maybe it could be bundled)

Reply:
Would we just want to display the output CSV file like [this](https://help.tower.nf/22.1/reports/overview/#visualising-reports)?